Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/sparkjobview.js

```diff
@@ -265,7 +265,7 @@ function getSparkDriverLog() {
         executorsObject = JSON.parse(s);
         var hostPort = getDriverPortFromExecutor(executorsObject);
         ipAddress = hostPort.split(":")[0];
-        var url = localhost + projectId + "/jobhistory/logs/" + ipAddress + "/port/30050/" + containerId + "/" + containerId + "/spark/stderr?restType=yarnhistory";
+        var url = localhost + projectId + "/jobhistory/logs/" + ipAddress + "/port/30050/" + containerId + "/" + containerId + "/livy/stderr?restType=yarnhistory";
```

Comment:
I remember the driverUrl can get from Livy response rather than combin it that is always changed without notifications.